ci/daily-snapshot.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+pr: none
+trigger: none
+schedules:
+- cron: "0 3 * * *"
+  displayName: Daily split snapshot
+  branches:
+    include:
+    - main
+
+jobs:
+- job: check_for_release
+  dependsOn:
+    - git_sha
+  variables:
+    branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
+    fork_sha: $[ dependencies.git_sha.outputs['out.fork_point'] ]
+  pool:
+    name: "ubuntu_20_04"
+    demands: assignment -equals default
+  steps:
+    - template: bash-lib.yml
+      parameters:
+        var_name: bash-lib
+    - bash: |
+        set -euo pipefail
+        eval "$(./dev-env/bin/dade-assist)"
+        source $(bash-lib)
+
+        release=$(./release.sh snapshot HEAD 0.0.0 | awk '{print $2}')
```

Comment:
I still feel slightly uneasy about this but don’t really have a better idea for now so let’s go with that.

Reply:
The `0.0.0` prefix? Would you prefer something smarter like:
```
$(head -1 LATEST | awk '{print $2}' | sed -e 's/\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/')
```

Or just the general idea of having daily snapshots?

Reply:
Yeah it’s about the prefix. I kind of liked seeing very easily where a snapshot fits in before and while I can figure it out based on commit numbers & hashes, it’s a lot less convenient for humans.